command/applications/application_save.go

```diff
@@ -20,7 +20,13 @@ func (f *flagStringArray) String() string {
 }
 
 func (f *flagStringArray) Set(value string) error {
-	*f = strings.Split(value, ",")
+	stripped := make([]string, 0)
```

Comment:
Why only for application_save?

Reply:
Not used for any others at this point, should I just factor these out into a lib at this point?

Reply:
Up to you, was just curious what the pattern was